packages/airnode-node/src/handlers/process-http-signed-data-request.ts

```diff
@@ -17,29 +19,43 @@ export async function processHttpSignedDataRequest(
 
   const endpoints = find(config.ois, ['title', trigger.oisTitle])?.endpoints;
   const endpoint = find(endpoints, ['name', trigger.endpointName]);
+  const decodedParameters = evm.encoding.safeDecode(parameters);
```

Comment:
I think it would be slightly better to do this after the `if (!endpoint)` check? i.e. above the `if (!decodedParameters)` part